src/Routes/ImageManager/Images.js

```diff
@@ -275,6 +275,13 @@ const Images = () => {
     setSortBy({ index, direction });
   };
 
+  const tableRowActions = [
+    {
+      title: 'Download',
+      onClick: (event, rowId, rowData) => window.open(rowData.isoURL),
```

Comment:
What happens if `isoURL` is undefined? Can you disable this action and show a tooltip saying this?
`window.open` is fine but I think setting `title` to `a` (anchor element) is better and easier to test later.